### ISSUE TITLE
Abandon package php-cs-fixer/diff

### ISSUE THREAD

**Issue by LocalHeroPro** [nunomaduro/phpinsights#598]
| Q                | A
| ---------------- | -----
| Bug report?      | yes/no
| Feature request? | no
| Library version  | 2.5.0

Run in terminal:
```bash
$ composer require nunomaduro/phpinsights
//
Package php-cs-fixer/diff is abandoned, you should avoid using it. No replacement was suggested.
//
```

**Comment by cmgmyr** [comment]
Yes, this package is recently abandoned. I should be able to update the dependency this week. Thank you for reporting

**Comment by cmgmyr** [comment]
Fixed in `v2.6.0`
